components/form/FirstStep.tsx

```diff
@@ -14,8 +26,9 @@ function FirstStep({ onfidoInstance, onSubmit, loading }: { onfidoInstance: SdkH
       <h3 className="mb-4">We want to get to know you!</h3>
       <p>Start by introducing yourself here.</p>
       <p>On the next page, we&apos;ll ask you to provide other information (documents or photos) that will help verify your identity.</p>
+      <p>Keep in mind that applicant under 18 cannot be verified.</p>
```

Comment:
I'd change "Keep in mind that applicant under 18 cannot be verified." to "Note: applicants younger than 18 years old will not be verified."

Reply:
Fixed!